hack/generate-role-yaml.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+
+import sys
+import yaml
+
+if len(sys.argv) == 1:
+    print(" Usage: %s <input yaml file>..." % sys.argv[0])
+    exit(1)
+
+roles = {}
+nrurls = {}
+for f in sys.argv[1:]:
+    s = open(f, "r")
+    docs = yaml.load_all(s, Loader=yaml.Loader)
```

Comment:
nit:
```
with open(f, "r") as  s:
     docs = yaml.load_all(s, Loader=yaml.Loader)

[...]
```
I don't think you need to explicitly call close() at the end

Reply:
Thanks, fixed.